loss-detection-app/res/device-mqtt-go/configuration.toml

```diff
@@ -1,113 +1,157 @@
+MaxEventSize = 0 # value 0 unlimit the maximum event size that can be sent to message bus or core-data
```

Comment:
Why is this file still here? As we discussed it can now be removed since it is identical to the one in the device-mqtt container.